libtrap/src/ifc_tls.c

```diff
@@ -1793,8 +1803,9 @@ int create_tls_sender_ifc(trap_ctx_priv_t *ctx, const char *params, trap_output_
    priv->backup_buffer = (void *) calloc(1, priv->int_mess_header.data_length +
                            sizeof(trap_buffer_header_t));
 
-   if (priv->clients == NULL) {
+   if (priv->backup_buffer == NULL) {
       /* if some memory could not have been allocated, we cannot continue */
+      result = TRAP_E_MEMORY;
```

Comment:
Here should probably be some error message, but I do not know which one. Please suggest and I will fix it.

My suggestion is:  VERBOSE(CL_ERROR, "Memory allocation for backup_buffer failed.");

Reply:
I think returnng TRAP_E_MEMORY should be enough.